components/raft_log_engine/src/engine.rs

```diff
@@ -74,8 +74,12 @@ impl RaftLogBatchTrait for RaftLogBatch {
         Ok(())
     }
 
+    #[inline]
     fn persist_size(&self) -> usize {
-        panic!("persist_size is not implemented for raft engine");
+        // TODO: This is not data size, but sufficient for current usage.
```

Comment:
/cc @tabokie as a reminder.

Reply:
It was implemented in https://github.com/tikv/raft-engine/pull/82 a few days ago.